docs/quickstart/work-with-data-locally.md

```diff
@@ -0,0 +1,142 @@
+---
+title: 7️⃣ Work with lakeFS data locally
+description: lakeFS quickstart / Bring lakeFS data to a local environment to show how lakeFS can be used for ML experiments development. 
+parent: ⭐ Quickstart
+nav_order: 35
+next: ["Resources for learning more about lakeFS", "./learning-more-lakefs.html"]
+previous: ["Using Actions and Hooks in lakeFS", "./actions-and-hooks.html"]
+---
+
+# Work with lakeFS Data Locally
+
+When working with lakeFS, there are scenarios where we need to access and manipulate data locally. An example use case for working 
+locally is machine learning model development. Machine learning model development is dynamic and iterative. To optimize this 
+process, experiments need to be conducted with speed, tracking ease, and reproducibility. Localizing model data during development 
+accelerates the process by enabling interactive and offline development and reducing data access latency.
+
+We're going to use [lakectl local](../howto/local-checkouts.md) to bring a subset of our lakeFS data to a local directory within the lakeFS
+container and edit an image dataset used for ML model development.
+
+## Cloning a Subset of lakeFS Data into a Local Directory
+
+1. In lakeFS create a new branch called `my-experiment`. You can do this through the UI or with `lakectl`:
+
+    ```bash
+    docker exec lakefs \
+        lakectl branch create \
+            lakefs://quickstart/my-experiment \
+            --source lakefs://quickstart/main
+    ```
+
+2. Clone images from your quickstart repository into a local directory named `my_local_dir` within your container:   
+
+    ```bash
+    docker exec lakefs \
```

Comment:
since the tutorial is self contained in the docker container, perhaps do a single

```
docker exec -it lakefs bash
```

and explain all other commands should execute there?